integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java

```diff
@@ -197,8 +197,6 @@ private static List<String> getJsonPath(String keyPath) {
   }
 
   private static boolean isArrayType(String value) {
-    return value.startsWith(ARRAY_PREFIX_CHAR)
-        && value.endsWith(ARRAY_SUFFIX_CHAR)
-        && value.contains(DISABLED_FACETS_SEPARATOR);
```

Comment:
Why was this removed?

Reply:
problem with this are single element arrays which do ton contain `;` separator